src/coreclr/jit/importercalls.cpp

```diff
@@ -2589,7 +2590,7 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
             case NI_System_Runtime_CompilerServices_RuntimeHelpers_IsKnownConstant:
 
             // We need these to be able to fold "typeof(...) == typeof(...)"
-            case NI_System_RuntimeTypeHandle_ToIntPtr:
+            case NI_System_RuntimeType_get_TypeHandle:
```

Comment:
`NI_System_RuntimeType_get_TypeHandle` should not be involved in `typeof(...) == typeof(...)` folding.

Should `NI_System_RuntimeType_get_TypeHandle` rather be in the lightweight intrinsics set below?

Reply:
Addressed